plugins/javadoc/src/main/resources/views/class.korte

```diff
@@ -163,29 +169,31 @@
                                 <!--   -->
                             </a>
                                 <h3>Method Summary</h3>
-                                <table class="memberSummary">
-                                    <caption><span id="t0" class="activeTableTab"><span>All Methods</span><span
-                                            class="tabEnd">&nbsp;</span></span><span id="t1" class="tableTab"><span><a
-                                            href="javascript:show(1);">Static Methods</a></span><span class="tabEnd">&nbsp;</span></span><span
-                                            id="t4" class="tableTab"><span><a href="javascript:show(8);">Concrete Methods</a></span><span
-                                            class="tabEnd">&nbsp;</span></span></caption>
-                                    <tbody>
-                                    <tr>
-                                        <th class="colFirst" scope="col">Modifier and Type</th>
-                                        <th class="colSecond" scope="col">Method</th>
-                                        <th class="colLast" scope="col">Description</th>
-                                    </tr>
-                                    {% for method in methods.own %}
-                                    <tr id="i{{ loop.index0 }}" class="{{ rowColor(loop.index0) }}">
-                                        <td class="colFirst"><code>{{ method.signature.modifiers|raw }}</code>
-                                        </td>
-                                        <th class="colSecond" scope="row"><code>{{ method.signature.signatureWithoutModifiers|raw }} </code>
-                                        </th>
-                                        <td class="colLast">{{ method.brief|raw }}</td>
-                                    </tr>
-                                    {% endfor %}
-                                    </tbody>
-                                </table>
+                                <div class="memberSummary">
+                                    <div role="tablist" aria-orientation="horizontal">
+                                        <button role="tab" aria-selected="true" aria-controls="memberSummary_tabpanel" tabindex="0" onkeydown="switchTab(event)" id="t0" class="activeTableTab">All Methods</button>
+                                        <button role="tab" aria-selected="false" aria-controls="memberSummary_tabpanel" tabindex="-1" onkeydown="switchTab(event)" id="t2" class="tableTab" onclick="show(2);">Instance Methods</button>
```

Comment:
if you will have any more fixes to this PR i'd just leave `All Methods` only (and remove rest) since we don't have a logic prepared to handle that (both on frontend and backend). Issue: https://github.com/Kotlin/dokka/issues/1118

Reply:
Done